client/src/components/pages/Home/index.css

```diff
@@ -0,0 +1,54 @@
+.home {
+    padding-top: 0rem;
+}
+
+.home__header {
+    margin-top: 3.5rem;
+}
+
+.home__header__img__overlay {
+    background: url('https://images.pexels.com/photos/710743/pexels-photo-710743.jpeg') no-repeat ;
```

Comment:
don't use external images for your app, it's out of your control and can get removed any time.. Put it into your repo instead (of course be careful about copyrights ;)
Btw this image has ~9 MB, not a very good idea to use it as a background. It can be [optimized](https://www.abetterlemonadestand.com/optimizing-images-for-web/), but don't waste time with that for now, just put it into your repo..

Reply:
Done ^_^